src/parameter/parameterslider.cpp

```diff
@@ -13,14 +13,21 @@ ParameterSlider::ParameterSlider(ParameterObject *parameterobject, int showDescr
 	connect(slider, SIGNAL(sliderReleased()), this, SLOT(onReleased()));
 	connect(slider, SIGNAL(valueChanged(int)), this, SLOT(onSliderChanged(int)));
 	connect(doubleSpinBox, SIGNAL(valueChanged(double)), this, SLOT(onSpinBoxChanged(double)));
-	if (showDescription == 0) {
+	if (showDescription == 0 || showDescription == 3) {
```

Comment:
This may be a good refactoring for later: Use an enum instead of the magic `showDescription` integer

Reply:
entered into #2316